packages/chaire-lib-backend/src/services/routing/Routing.ts

```diff
@@ -69,11 +69,13 @@ const prepareResults = (
                     ? (walkingRouteResult.result as RouteResults).routes[0]
                     : undefined;
 
+            const hasAlternativeWalkPath = walkOnlyPath ? true : false;
```

Comment:
Maybe a comment here, I'm not sure what we mean.

Reply:
Même avec le commentaire, c'est pas clair. Et on ne devrait pas faire référence au comparison tab ici. C'est le backend. à ce point-ci (ie dans le backend), est-ce que le fait que le walkOnlyPath ne soit pas `undefined` est suffisant? Ensuite, le frontend peut gérer ce champ comme il veut pour ajouter le hasAlternativeWalkPath à la bonne place